lib/widgets/comment.dart

```diff
@@ -48,48 +49,58 @@ class Comment extends HookWidget {
   _showCommentInfo(BuildContext context) {
     final com = commentTree.comment;
     showInfoTablePopup(context, {
-      'id': com.id,
-      'creatorId': com.creatorId,
-      'postId': com.postId,
-      'postName': com.postName,
-      'parentId': com.parentId,
-      'removed': com.removed,
-      'read': com.read,
-      'published': com.published,
-      'updated': com.updated,
-      'deleted': com.deleted,
-      'apId': com.apId,
-      'local': com.local,
-      'communityId': com.communityId,
-      'communityActorId': com.communityActorId,
-      'communityLocal': com.communityLocal,
-      'communityName': com.communityName,
-      'communityIcon': com.communityIcon,
-      'banned': com.banned,
-      'bannedFromCommunity': com.bannedFromCommunity,
-      'creatorActirId': com.creatorActorId,
-      'userId': com.userId,
-      'upvotes': com.upvotes,
-      'downvotes': com.downvotes,
-      'score': com.score,
-      '% of upvotes': '${100 * (com.upvotes / (com.upvotes + com.downvotes))}%',
-      'hotRank': com.hotRank,
-      'hotRankActive': com.hotRankActive,
+      'id': com.comment.id,
+      'creatorId': com.comment.creatorId,
+      'postId': com.comment.postId,
+      'postName': com.post.name,
+      'parentId': com.comment.parentId,
+      'removed': com.comment.removed,
+      'read': com.comment.read,
+      'published': com.comment.published,
+      'updated': com.comment.updated,
+      'deleted': com.comment.deleted,
+      'apId': com.comment.apId,
+      'local': com.comment.local,
+      'communityId': com.community.id,
+      'communityActorId': com.community.actorId,
+      'communityLocal': com.community.local,
+      'communityName': com.community.name,
+      'communityIcon': com.community.icon,
+      'banned': com.creator.banned,
+      'bannedFromCommunity': com.creatorBannedFromCommunity,
+      'creatorActirId': com.creator.actorId,
+      'userId': com.creator.id,
+      'upvotes': com.counts.upvotes,
+      'downvotes': com.counts.downvotes,
+      'score': com.counts.score,
+      '% of upvotes':
+          '${100 * (com.counts.upvotes / (com.counts.upvotes + com.counts.downvotes))}%',
     });
   }
 
-  bool get isOP => commentTree.comment.creatorId == postCreatorId;
-  bool get isMine =>
-      commentTree.comment.creatorId == commentTree.comment.userId;
+  bool get isOP =>
+      commentTree.comment.comment.creatorId ==
+      commentTree.comment.post.creatorId;
+
+  // bool get isMine => commentTree.comment.comment.creatorId == ;
```

Comment:
I think what you implemented is the best we can do, you can remove those comments